src/item.cpp

```diff
@@ -4621,16 +4623,55 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
                   contents_item.charges > 1 )
                 ? contents_item.charges
                 : 1;
+
+            // Color second half of container contents depending on perishability (highest perishability of all contents)
+            if( contents_item.is_food() ) {
+                // For comestibles
+				const item_pocket* const parent_pocket = contained_where(contents_item);
+                colorprefix = "<" + string_from_color( contents_item.color_in_inventory() ) + ">";
+                colorprefix.replace( 1, 1, "color" ); // changes <c_cyan> to <color_cyan>
+                if( contents.get_sealed_summary() == item_contents::sealed_summary::all_sealed && parent_pocket->spoil_multiplier() == 0 ) {
```

Comment:
And it seems this only checks for sealed container but not rot progress, whereas the code below in the multiple-item case only checks for rot progress but not sealed container?

Reply:
The top checks for rot progress by calling contents_item.color_in_inventory(), although sealed will supercede it.
The bottom only currently checks rot progress without checking sealed since multiple items in sealed containers isn't a thing right now(?) (see lower comment, if not I can put it in!)
And yes, I think checking sealed and the parent pocket are redundant, the parent pocket by itself is probably (?) fine, will fix!